### ISSUE TITLE
Create a tag whenever the version number is updated?

### ISSUE THREAD

**Issue by agarny** [cmlibs/zinc#79]
I had to rebuild the Zinc library on Ubuntu, which resulted in my (very basic) tests to fail because of the version number having been increased to `3.1.2` from `3.1.1`. So, I was wondering whether a tag could be created for `3.1.2`? Then again, I am building Zinc through [`OpenCMISS-setup`](https://github.com/OpenCMISS/setup), so I am not sure how that would work...

**Comment by hsorby** [comment]
I agree that there should at least be a tag when updating the version number.  Technically I think we should make a commit to the master/release branch, I thought that was what we decided when we choose this development paradigm.
